### ISSUE TITLE
K3s does not work with nats 2.8.3

### ISSUE THREAD

**Issue by brandond** [k3s-io/kine#122]
Unsure what the issue is, but K3s fails to come up when using `docker.io/library/nats:2.8.3`; `docker.io/library/nats:2.7.4` seems to work fine as does `docker.io/library/nats:2.8.2`

The terminal error is:
```
FATA[0064] failed to start controllers: failed to create new server context: failed to register CRDs: the server was unable to return a response in the time allotted, but may still be processing the request (get customresourcedefinitions.apiextensions.k8s.io)
```

I am just using the nats docker image: `docker run --rm -it docker.io/library/nats:2.8.3 -js`

**Comment by brandond** [comment]
cc @boxboatmatt @jnmoyne @caleblloyd

**Comment by caleblloyd** [comment]
NATS 2.8.3 had an issue where KV Watchers would hang

I would recommend using 2.8.2 for now, and it will be fixed in the upcoming 2.8.4. PR for the fix has already been merged:

https://github.com/nats-io/nats-server/pull/3148

**Comment by brandond** [comment]
Glad to hear it! cc @VestigeJ
